src/main/java/frc/robot/commands/SetShooterSpeedAndWait.java

```diff
@@ -4,26 +4,26 @@
 
 package frc.robot.commands;
 
-import java.util.function.DoubleSupplier;
+import java.util.function.Supplier;
 
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.Shooter;
 
 public class SetShooterSpeedAndWait extends Command {
     private final Shooter m_shooter;
-    private final DoubleSupplier m_leftRpm, m_rightRpm;
+    private final Supplier<Shooter.ShooterValues> m_valueSupplier;
 
-    public SetShooterSpeedAndWait(Shooter shooter, DoubleSupplier leftRpm, DoubleSupplier rightRpm) {
+    public SetShooterSpeedAndWait(Shooter shooter, Supplier<Shooter.ShooterValues> valueSupplier) {
```

Comment:
Since this command actually calls the Shooter.setShooterRpms() method, it should be called SetShooterRpmsAndWait. There is a separate Shooter.setShooterSpeeds. We want to avoid confusion.